src/main/java/appeng/me/storage/CellInventory.java

```diff
@@ -43,38 +44,40 @@
 import appeng.api.storage.data.IItemList;
 import appeng.util.Platform;
 import appeng.util.item.AEItemStack;
+import appeng.util.item.UnsortedItemList;
 
 
 public class CellInventory implements ICellInventory
 {
 
-	static final String ITEM_TYPE_TAG = "it";
-	static final String ITEM_COUNT_TAG = "ic";
-	static final String ITEM_SLOT = "#";
-	static final String ITEM_SLOT_COUNT = "@";
-	static final String ITEM_PRE_FORMATTED_COUNT = "PF";
-	static final String ITEM_PRE_FORMATTED_SLOT = "PF#";
-	static final String ITEM_PRE_FORMATTED_NAME = "PN";
-	static final String ITEM_PRE_FORMATTED_FUZZY = "FP";
-	private static final HashSet<Integer> BLACK_LIST = new HashSet<Integer>();
-	protected static String[] itemSlots;
-	protected static String[] itemSlotCount;
-	protected final NBTTagCompound tagCompound;
-	protected final ISaveProvider container;
-	protected int maxItemTypes = 63;
-	protected short storedItems = 0;
-	protected int storedItemCount = 0;
-	protected IItemList<IAEItemStack> cellItems;
-	protected ItemStack i;
-	protected IStorageCell CellType;
-
-	protected CellInventory( NBTTagCompound data, ISaveProvider container )
+	private static final String ITEM_TYPE_TAG = "it";
+	private static final String ITEM_COUNT_TAG = "ic";
+	private static final String ITEM_SLOT = "#";
+	private static final String ITEM_SLOT_COUNT = "@";
+	private static final String ITEM_PRE_FORMATTED_COUNT = "PF";
+	private static final String ITEM_PRE_FORMATTED_SLOT = "PF#";
+	private static final String ITEM_PRE_FORMATTED_NAME = "PN";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "ITEM_PRE_FORMATTED_NAME" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)